Packs/SplunkPy/Integrations/SplunkPy/SplunkPy.py

```diff
@@ -2394,9 +2421,18 @@ def replace_keys(data):
     return data
 
 
-def kv_store_collection_create(service: client.Service) -> None:
-    service.kvstore.create(demisto.args()['kv_store_name'])
-    return_outputs("KV store collection {} created successfully".format(service.namespace['app']), {}, {})
+def kv_store_collection_create(service: client.Service, args: dict) -> CommandResults:
+    try:
+        service.kvstore.create(args['kv_store_name'])
+    except HTTPError as error:
+        if error.status == 409 and error.reason == 'Conflict':
+            raise DemistoException(
+                f"KV store collection {service.namespace['app']} already exists.",
+            ) from error
+
+    return CommandResults(
+        readable_output=f"KV store collection {service.namespace['app']} created successfully",
+    )
```

Comment:
looks good! 
but from where is it?

Reply:
when i trying this command (create collection that already exists ),
 i got unclear error, so , i added it...